### ISSUE TITLE
Install fail with pytorch3dunet=1.2.5

### ISSUE THREAD

**Issue by lorenzocerrone** [hci-unihd/plant-seg#33]
```
Traceback (most recent call last):
  File "/home/lcerrone_local/miniconda3/envs/plant-seg/bin/plantseg", line 7, in <module>
    from plantseg.run_plantseg import main
  File "/home/lcerrone_local/miniconda3/envs/plant-seg/lib/python3.7/site-packages/plantseg/run_plantseg.py", line 3, in <module>
    from plantseg.pipeline.raw2seg import raw2seg
  File "/home/lcerrone_local/miniconda3/envs/plant-seg/lib/python3.7/site-packages/plantseg/pipeline/raw2seg.py", line 5, in <module>
    from plantseg.predictions.predict import UnetPredictions
  File "/home/lcerrone_local/miniconda3/envs/plant-seg/lib/python3.7/site-packages/plantseg/predictions/predict.py", line 6, in <module>
    from pytorch3dunet.datasets.hdf5 import get_test_loaders
ImportError: cannot import name 'get_test_loaders' from 'pytorch3dunet.datasets.hdf5' (/home/lcerrone_local/miniconda3/envs/plant-seg/lib/python3.7/site-packages/pytorch3dunet-1.2.5-py3.7.egg/pytorch3dunet/datasets/hdf5.py)

```

**Comment by lorenzocerrone** [comment]
Last working version is 1.2.4, for now I will just change the readme

**Comment by lorenzocerrone** [comment]
fix in version 1.0.9
